gloo/docs/operator_guide/observability.md

```diff
@@ -1,41 +1,88 @@
 ---
 title: Observability
 weight: 3
+description: How to monitor and trace withing your Gloo setup.
 ---
 
+All Gloo pods ship with optional [Prometheus](https://prometheus.io/) monitoring capabilities.
 
-All gloo pods ship with optional [prometheus](https://prometheus.io/) monitoring as well as [open tracing](https://opentracing.io/) capability.
+This functionality is turned off by default, and can be turned on a couple of different ways: through [Helm chart install
+options]({{< ref "/installation/kubernetes/quick_start#install_helm" >}}); and through environment variables.
 
-This functionality is turned off by default but can be turned on a couple of different ways.
+### Helm Chart Options
 
 The first way is via the helm chart. All deployment objects in the helm templates accept an argument `stats` which
-when set to true, start a stats server on the given pod. for example: 
-```yaml
-gloo:
-  deployment:
-    image:
-      repository: quay.io/solo-io/gloo
-      pullPolicy: Always
-    xdsPort: 9977
-    replicas: 1
-    stats: true
+when set to true, start a stats server on the given pod.
+
+For example, to add stats to the Gloo `gateway`, when installing with Helm add  `--set gateway.deployment.stats=true`.
+The following example shows how to enable stats for both gateway and discovery.
+
+```shell
+helm install gloo/gloo \
+  --name gloo \
+  --namespace gloo-system \
+  --set gateway.deployment.stats=true,discovery.deployment.stats=true
```

Comment:
It's a bit hard to discern that we're setting stats true to two components, can you split it up into two `--set` flags? for example `--set gateway.deployment.stats=true --set discovery.deployment.stats=true`

Reply:
I simplified to enabling stats on a single service, and updated other references